broker-cli/proto/broker.proto

```diff
@@ -847,4 +863,27 @@ service WalletService {
    * ```
    */
   rpc CommitBalance (CommitBalanceRequest) returns (google.protobuf.Empty);
+  /**
+   * GetPaymentChannelNetworkAddress retrieves the Payment Channel Network Address for the given currency.
+   *
+   * ```javascript
+   * const { paymentChannelNetworkAddress } = await walletService.getPaymentChannelNetworkAddress({ symbol: 'BTC' })
+   * console.log(paymentChannelNetworkAddress)
+   * ```
+   *
+   * ```shell
+   * > sparkswap wallet network-address BTC
+   * ```
+   *
+   * > The above command will print:
+   *
+   * ```javascript
+   * > "025f08036e00e38468f52d68bb83939995440b72730bfaae99ace5630630997623@relayer.testnet.sparkswap.com:10113"
```

Comment:
[nit] would change the documented IP to be `your.daemon:10113` instead of relayer